handler.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/dgrijalva/jwt-go"
```

Comment:
please use https://github.com/golang-jwt/jwt, dgrijalva/jwt-go is deprecated and has some security vulnerability